src/components/common/GitInfoMaterial.tsx

```diff
@@ -29,9 +30,11 @@ export default function GitInfoMaterial({
     hideWebhookModal,
     workflowId,
     onClickShowBranchRegexModal,
+    appId,
```

Comment:
use useParams instead of passing app Id.

Reply:
done